linter.ini

```diff
@@ -2,6 +2,14 @@
 ignore = E252,W504,W503
 max-line-length = 120
 
+per-file-ignores=
+    setup.py:E402
+    ./eth2spec/phase0/*.py:E3,E402
+    ./eth2spec/altair/*.py:E3,E402
+    ./eth2spec/merge/*.py:E3,E402
```

Comment:
I'm a little concerned about ignoring `E3XX` since we'd like to check `E303` in the markdown files, inside the functions.

Reply:
Fixed. I was being lazy.